File: go/chapter6/pets.go
package main

import "fmt"

func main() {
	pets := map[string]map[string]interface{}{
		"sparrow": {
			"animal": "Bird",
			"owner":      "Sparrow",
		},
		"bolt": {
			"animal": "Dog",
			"owner":      "Sparrow",
		},
	}

	for key, value := range pets {
		fmt.Printf("%s\n", key)
		for k, v := range value {
			fmt.Printf("%s: %v\n", k, v)
		}
		fmt.Println()
	}
}

File: python/chapter6/pets.py
pets = {
    "sparrow": {
        "animal": "Bird",
        "owner": "Sparrow",
    },
    "bolt": {
        "animal": "Dog",
        "owner": "Sparrow",
    },
}

for key, value in pets.items():
    print("%s\n", key)

    for k, v in value.items():
        print(f"{k}: {v}")

    print()
